packages/common/src/services/logging/transporters/ConsoleTransporter.ts

```diff
@@ -0,0 +1,29 @@
+import { LogLevel } from '../LogLevel';
+
+import type LogTransporter from './LogTransporter';
+
+export default class ConsoleTransporter implements LogTransporter {
+  constructor(readonly logLevel: LogLevel) {}
+
+  log(level: LogLevel, scope: string, message: string, extra?: Record<string, unknown>, error?: Error) {
+    if (level < this.logLevel) return;
+
+    switch (level) {
+      case LogLevel.ERROR:
+      case LogLevel.FATAL:
+        console.error(`[${scope}] ${message}`, error);
```

Comment:
Should we prepend the log level to give more insight?
```suggestion
        console.error(`${level}: [${scope}] ${message}`, error);
```

Reply:
Sure! I removed the level to keep the messages cleaner, but I'm not against re-adding it.